[2025-05-27T20:42:21.385+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_custom_dag.bash_says_hi manual__2025-05-27T20:42:19.594619+00:00 [queued]>
[2025-05-27T20:42:21.416+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_custom_dag.bash_says_hi manual__2025-05-27T20:42:19.594619+00:00 [queued]>
[2025-05-27T20:42:21.418+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-05-27T20:42:21.455+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): bash_says_hi> on 2025-05-27 20:42:19.594619+00:00
[2025-05-27T20:42:21.466+0000] {standard_task_runner.py:60} INFO - Started process 578 to run task
[2025-05-27T20:42:21.473+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'first_custom_dag', 'bash_says_hi', 'manual__2025-05-27T20:42:19.594619+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/__pycache__/first_dag.py', '--cfg-path', '/tmp/tmp3ka04fk3']
[2025-05-27T20:42:21.478+0000] {standard_task_runner.py:88} INFO - Job 2: Subtask bash_says_hi
[2025-05-27T20:42:21.603+0000] {task_command.py:423} INFO - Running <TaskInstance: first_custom_dag.bash_says_hi manual__2025-05-27T20:42:19.594619+00:00 [running]> on host ab9400de9a7e
[2025-05-27T20:42:21.831+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_custom_dag' AIRFLOW_CTX_TASK_ID='bash_says_hi' AIRFLOW_CTX_EXECUTION_DATE='2025-05-27T20:42:19.594619+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-27T20:42:19.594619+00:00'
[2025-05-27T20:42:21.835+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-27T20:42:21.837+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello from Bash!'"]
[2025-05-27T20:42:21.862+0000] {subprocess.py:86} INFO - Output:
[2025-05-27T20:42:21.890+0000] {subprocess.py:93} INFO - Hello from Bash!
[2025-05-27T20:42:21.893+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-27T20:42:21.953+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=first_custom_dag, task_id=bash_says_hi, execution_date=20250527T204219, start_date=20250527T204221, end_date=20250527T204221
[2025-05-27T20:42:22.009+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-27T20:42:22.060+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
